bl-kernel/admin/themes/booty/css/bludit.css

```
@@ -827,3 +827,29 @@ select:focus,
 }
+
+/* Highlight the table row pointed by the URL anchor, e.g. after activating a plugin */
+.table > tbody > tr:target > td {
+	animation: rowHighlight 2.5s ease-out forwards;
+}
+
+.table > tbody > tr:target {
+	scroll-margin-top: 20px;
+}
+
+@keyframes rowHighlight {
```

Stylelint (17.14.0)

[error] Expected keyframe name "rowHighlight" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)